dashboard/vendor/assets/javascripts/jquery.overlaps.js

```diff
@@ -0,0 +1,71 @@
+/*! Copyright (c) 2010 Brandon Aaron (http://brandonaaron.net)
+ * Licensed under the MIT License (LICENSE.txt).
+ *
```

Comment:
I wouldn't mind a comment-only modification like that (the license certainly allows it  and there is no harm if it gets lost with a version update), but if you have concerns it's also fine not to do that.